polyfills/Element/prototype/cloneNode/detect.js

```diff
@@ -1,6 +1,8 @@
 'document' in this && "cloneNode" in document.documentElement && (function() {
-	var test = document.createElement('input');
+	var div = document.createElement('div'), test = document.createElement('input');
 	test.checked = true;
+	div.appendChild(test);
 	var result = test.cloneNode();
-	return !!result.checked;
+	var result2 = div.cloneNode();
```

Comment:
I have pushed the updated code.

Just to be sure, I downloaded Firefox 28 and tested the detection code and polyfill. It is working as expected.